core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -1817,7 +1817,7 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
                 addr = GET_OPERAND(uint32, I32, 2);
                 frame_ip += 4;
                 CHECK_MEMORY_OVERFLOW(1);
-                *(uint8 *)maddr = (uint8)sval;
+                STORE_U8(maddr, (uint8_t)sval);
```

Comment:
Wondering why the data race only occurs in fast-interp? Should we also modify the similar line in classic-interpter:
https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/iwasm/interpreter/wasm_interp_classic.c#L2081

Reply:
Hi
At the moment we want to enable tsan for fast-interp and aot 

Changing that doesn't really fix anything but just allows such a suppression to happen correctly

Reply:
Got it, thanks.